src/script/microbit-interfacing/MicrobitConnection.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
```

Comment:
Should this be copyright the Foundation?

Reply:
Let's pick this discussion up offline. It's far from the only new source file to use the existing notice.